accepted/future-releases/3009-inference-using-bounds/feature-specification.md

```diff
@@ -0,0 +1,319 @@
+# Inference using bounds
+
+Author: Chloe Stefantsova
+
+Status: Accepted
+
+Version 0.1 (see [CHANGELOG](#CHANGELOG) at end)
```

Comment:
Personally I would start at 1.0. To me, "0.1" means it's a draft, and considering that we've already pretty much committed to doing this feature, it seems like we're well out of "draft" territory 😃

Reply:
Thanks for the suggestion. I've updated the file accordingly.